rust_bindings/build.rs

```diff
@@ -8,9 +8,7 @@ fn main() {
     println!("cargo:rerun-if-changed=wrapper.h");
     println!("cargo:rerun-if-changed=../src/c_bindings/c_wrapper.h");
     println!("cargo:rerun-if-changed=../src/c_bindings/c_wrapper.cpp");
-
-    // Link GMP library using pkg-config
-    pkg_config::Config::new().probe("gmp").unwrap();
+    println!("cargo:rustc-link-lib=gmp");
```

Comment:
is this linking statically? I imagine it won't work if we link against a system-installed GMP and then hope that the user has the same installed.

Reply:
It shouldn't be static